tigris/collection.go

```diff
@@ -193,6 +193,22 @@ func (c *Collection[T]) Read(ctx context.Context, filter filter.Filter, fields .
 	return &Iterator[T]{Iterator: it}, nil
 }
 
+// Read returns documents which satisfies the filter.
+// Only field from the give fields are populated in the documents. By default, all fields are populated.
+func (c *Collection[T]) Explain(ctx context.Context, filter filter.Filter, fields ...*fields.Read) (*driver.ExplainResponse, error) {
```

Comment:
For the fields which passed as objects via HTTP, we need to add them to scripts/fix_openapi.yaml. "filter" and "fields" in the case of explain API.
This will convert them to json.RawMessage for HTTP client generator.
And I just notices "filter" and "fields" were not added in tigris-api/scripts/fix_openapi.yaml, there we fix it from []byte to object.

Reply:
Nevermind. So as explain uses ReadRequest that doesn't apply.